api-tests/core/strapi/document-service/resources/schemas/author.js

```diff
@@ -10,15 +10,16 @@ module.exports = {
   draftAndPublish: true,
   pluginOptions: {
     i18n: {
-      localized: false,
+      localized: true,
     },
   },
   attributes: {
     name: {
       type: 'string',
+      unique: true,
       pluginOptions: {
         i18n: {
-          localized: false,
```

Comment:
other tests depend on this to be non localized , would it be possible to use another schema here? (category) 
Maybe I should add a comment actually mentioning this 🤔